components/net/fetch/cors_cache.rs

```diff
@@ -112,7 +112,7 @@ pub struct BasicCORSCache(Vec<CORSCacheEntry>);
 fn match_headers(cors_cache: &CORSCacheEntry, cors_req: &CacheRequestDetails) -> bool {
     cors_cache.origin == cors_req.origin &&
         cors_cache.url == cors_req.destination &&
```

Comment:
Hmm... spec says "url is request's current url", so I'd expect to see something like `cors_cache.url == cors_req.current_url()` instead.